pins:
    team_switch: 20
    team_feedback_led: 19
    reset_button: 16
